src/test/ui/nll/borrowed-universal-error-2.rs

```diff
@@ -11,12 +11,9 @@
 #![feature(nll)]
 #![allow(warnings)]
 
-fn gimme(x: &(u32,)) -> &u32 { &x.0 }
-
 fn foo<'a>(x: &'a (u32,)) -> &'a u32 {
     let v = 22;
-    let r = gimme(&(v,));
-    r
+    &v
```

Comment:
Output now:

```
lunch-box. rustc +nightly borrowed-universal-error-2.rs -Znll-dump-cause
error[E0597]: `v` does not live long enough
  --> borrowed-universal-error-2.rs:16:5
   |
16 |     &v
   |     ^^ borrowed value does not live long enough
17 | }
   | - borrowed value only lives until here
   |
   = note: borrowed value must be valid for lifetime '_#3r...
   = note: because of an outlives relation created at `bb0[4]`
           because of an outlives relation created at `bb0[5]`
           because of an outlives relation created at `bb0[5]`
           because of an outlives relation created at `bb0[0]`
           because `'_#1r` is universally quantified

error: aborting due to previous error
```

Still ungreat.